gulpfile.mjs

```diff
@@ -1930,6 +1930,12 @@ gulp.task("lint", function (done) {
     prettierOptions.push("--log-level", "warn", "--check");
   }
 
+  const svgLintOptions = [
+    "node_modules/svglint/bin/cli.js",
+    "**/*.svg",
```

Comment:
Does this also target e.g. the `node_modules/` folder, because if so it'd be nice if we could avoid that? As-is it seems that adding svglint here affects the overall run time of linting a bit.

(Note that ESLint and Prettier have ignore-files that handle this, which unfortunately doesn't *appear* to be available for svglint.)